addons/l10n_de/models/account_move.py

```diff
@@ -50,5 +50,6 @@ def _compute_l10n_de_addresses(self):
             elif record.move_type in ("in_invoice", "in_refund"):
                 data.append((_("Invoicing and Shipping Address:"), record.partner_id))
             else:
-                data.append((_("Shipping Address:"), record.partner_shipping_id))
+                if record.partner_shipping_id:
+                    data.append((_("Shipping Address:"), record.partner_shipping_id))
```

Comment:
To fix this, I think we should add a `or not record.partner_shipping_id` on line 50 so the "Shipping Address" is stille "defined".